orbs/e2e-web/orb.yml

```diff
@@ -0,0 +1,142 @@
+version: 2.1
+
+description: |
+  Run e2e web tests from e2e_jt_tests repo
+
+orbs:
+  service: "jobteaser/service@0.10.0"
+  slack: "circleci/slack@3.4.2"
+
+executors:
+  default:
+    parameters:
+      tags:
+        type: string
+        default: "not @flaky and not @pending and not @browserstack and not @prod"
+      max_instances:
+        type: string
+        default: "30"
+    resource_class: "xlarge"
+    docker:
+      - image: "circleci/openjdk:11.0.8-buster-node-browsers"
+        environment:
+          DEBUG: "false"
+          E2E_MAX_INSTANCES: << parameters.max_instances >>
+          # ensure we use the chromedriver embedded in the docker image
+          # rather than the one from package.json
+          CHROMEDRIVER_FILEPATH: /usr/local/bin/chromedriver
+          FORCE_COLOR: "0"
+          CUCUMBER_TAGS: << parameters.tags >>
+
+commands:
+  clone_test_suite:
+    description: "Clone test suite configuration"
+    steps:
+      - run:
+          name: "clone the test suite"
+          command: |
+            git clone git@github.com:jobteaser/e2e_jt_tests.git
+
+  manage_dependencies:
+    description: "Use cache management for dependencies or install fresh ones"
+    steps:
+      - restore_cache:
+          key: npm-dep-{{ checksum "e2e_jt_tests/package-lock.json" }}
+      - run:
+          working_directory: e2e_jt_tests
+          name: "install dependencies"
+          command: |
+            npm install
+      - save_cache:
+          key: npm-dep-{{ checksum "e2e_jt_tests/package-lock.json" }}
+          paths:
+            - ./e2e_jt_tests/node_modules
+
+  create_report_folder:
+    description: "Create reports directory"
+    steps:
+      - run:
+          working_directory: e2e_jt_tests
+          name: "create reports directory"
+          command: |
+            mkdir -p reports/screenshots
+            mkdir -p reports/webdriverServer
+
+  run_e2e_test:
+    parameters:
+      command:
+        description: "The command to run in e2e_jt_tests repo"
+        type: "string"
+        default: "npm run chrome"
+      e2e_cookie:
+        description: "The cookie to add to bypass rate limit"
+        type: "string"
+        default: $COOKIE_E2E
+    steps:
+      - run:
+          working_directory: e2e_jt_tests
+          name: "run the test suite with chrome on staging"
+#https://circleci.com/docs/2.0/configuration-reference/#default-shell-options
+# in case of failure, do not mark step as failed (experiment of e2e tests on this repo)
+          command: |
+            export BYPASS_RACK_ATTACK=<< parameters.e2e_cookie >>
+            << parameters.command >>
```

Comment:
**remark**

I am wondering if you could not hardcode `npm run chrome` here and only parameterize the eventual env vars that goes before it.

Reply:
So `parameters.command` would become `parameters.command_envs` for ex.

Reply:
We have several command like `npm run firefox` `npm run chrone:master` `npm run chrome:de`... I think it is better to let the command free to parameter in dev project depending on the need.